shared_workflow/shared_defaults.py

```diff
@@ -8,7 +8,7 @@
 global_root = workflow_config["global_root"]
 bin_process_dir = os.path.join(global_root, 'workflow/scripts')
 emod3d_version = workflow_config["emod3d_version"]
-params_vel = workflow_config['params_vel']
+vm_params = workflow_config['vm_params']
```

Comment:
if dependencies are correctly moved to qcore.constant, this shouldnt be needed anymore.